configs/mmcls/classification_vacc-fp16_static.py

```diff
@@ -0,0 +1,26 @@
+_base_ = ['./classification_static.py', '../_base_/backends/vacc.py']
+
+backend_config = dict(
+    common_config=dict(
+        vdsp_params_info=dict(
+            vdsp_op_type=300,
+            iimage_format=5000,
```

Comment:
typo: `iimage_format`, `...`

Reply:
Here is not an error. `iimage_format` is a param of vacc wrapper, which means 'input image format' setting.